setup.sh

```diff
@@ -64,105 +67,139 @@ then
   exit 1
 fi
 
-# github info for flux
-export ASB_GIT_REPO=$(git remote -v | cut -f 2 | cut -f 1 -d " " | head -n 1)
+# AAD admin group name
+if [ -z "$ASB_CLUSTER_ADMIN_GROUP" ]
+then
+  export ASB_CLUSTER_ADMIN_GROUP=cluster-admins-$ASB_TEAM_NAME
+fi
 
+# github info for flux
 if [ -z "$ASB_GIT_REPO" ]
 then
-  echo Please cd to an ASB git repo
-  exit 1
+  export ASB_GIT_REPO=$(git remote -v | cut -f 2 | cut -f 1 -d " " | head -n 1)
+
+  if [ -z "$ASB_GIT_REPO" ]
+  then
+    echo Please cd to an ASB git repo
+    exit 1
+  fi
+fi
+
+if [ -z "$ASB_GIT_PATH" ]
+then
+  export ASB_GIT_PATH=gitops
+fi
+
+if [ -z "$ASB_GIT_BRANCH" ]
+then
+  export ASB_GIT_BRANCH=$(git status  --porcelain --branch | head -n 1 | cut -f 2 -d " " | cut -f 1 -d .)
```

Comment:
I wondering if we should echo out the value of the variable after these conditionals so someone might catch it in-process and stop it if they've set a value that will eventually cause issues.

Reply:
it's in the prompt in Codespaces

I'll put a task on the board to display all the key values unless they pass in -y